### ISSUE TITLE
use 'roles_path' to centralize roles rather than symlinks

### ISSUE THREAD

**Issue by miabbott** [projectatomic/atomic-host-tests#39]
We currently symlink to the root `roles` directory from the test directories which use roles.  While this works, it has always seemed hacky.

The [roles_path](http://docs.ansible.com/ansible/intro_configuration.html#roles-path) variable could be used to centralize the location of the roles.  We would just need to make some adjustments to `ansible.cfg` on systems running the playbooks.

**Comment by miabbott** [comment]
I played around with this recently and wasn't able to come up with a solution that was flexible enough for our needs.

I used an `ansible.cfg` file that had the following:

```
[defaults]
roles_path=${PWD}/roles
```

This allowed me to remove the `roles` symlink from the various playbook directories under `tests`.  However, this forced the invocation of the playbooks to happen in the root of the repo:

```
$ cd atomic-host-tests
$ ansible-playbook -v -i inventory tests/improved-sanity-test/main.yml
```

This is not desirable (IMO), because it breaks the ability to do things like experiment from directly in a playbook directory:

```
$ cd atomic-host-tests/tests/new-playbook/
$ ansible-playbook -v -i inventory main.yml
```

...or invoke the playbook from outside the repo:

```
$ cd /home/miabbott
$ ansible-playbook -v -i inventory ~/source/projectatomic/atomic-hosts-tests/tests/improved-sanity-tests/main.yml
```

I'd rather keep the flexibility of being able to run the tests from anywhere, than imposing additional requirements on users of the tests.

Thoughts?
